kinder/ci/tools/update-workflows/templates/workflows/skew-kubelet-x-on-y.yaml

```diff
@@ -7,7 +7,7 @@ vars:
   kubeadmVersion: "\{\{ resolve `ci/{{ ciLabelFor .KubeadmVersion }}` \}\}"
   kubeletVersion: "\{\{ resolve `ci/{{ ciLabelFor .KubeletVersion }}` \}\}"
   kubernetesVersion: "\{\{ resolve `ci/{{ ciLabelFor .KubernetesVersion }}` \}\}"
-  ignorePreflightErrors: "KubeletVersion"
+  ignorePreflightErrors: "{{.IgnorePreflightErrors}}"
```

Comment:
i just realized this forces us to update the tools/update-workflows/config.yaml file on every new release, because the versions will shift. this is noisy / problematic.

how about we keep the `ignorePreflightErrors: "KubeletVersion"` hardcoded for all kubelet skews until we no longer need it ~3x releases from now?

i think unit tests in k/k should be enough to verify we support the N-3 skew, right?

Reply:
If so, we will ignore all KubeletVersion in the skew test here.

This will be easier to maintain. (The current change is more accurate to me. 🤣)

> i think unit tests in k/k should be enough to verify we support the N-3 skew, right?

Yes. This can be covered IIRC

Reply:
In this way, we have to wait to v1.32(probally) to remove all `ignorePreflightErrors: KubeletVersion` at that time.

Reply:
> In this way, we have to wait to v1.32(probally) to remove all `ignorePreflightErrors: KubeletVersion` at that time.

SGTM